src/terra-dev-site/ss/filler/PlaceholderPage.module.scss

```diff
@@ -0,0 +1,19 @@
+:local {
```

Comment:
Is it possible to test the scroll behavior without adding a bunch of test files?

Reply:
I have not been able to think of a better alternative. The original issue for this PR applies more to the Terra UI dev site. So on the plus side, adding a bunch of test files and directories does make it more like the Terra UI dev site.

Do you have any alternatives that I could look into trying out?

Reply:
I think to test this you'll have to add a bunch of files to enable secondary nav scrolling, but I also think you could make the files smaller and ditch the css with my mdx suggestion below.

Reply:
Done: https://github.com/cerner/terra-dev-site/pull/217/commits/c52489d70cfbf8073bc1839a448f19f3bef40f4d